grails-app/controllers/au/org/ala/specieslist/EditorController.groovy

```diff
@@ -95,13 +94,21 @@ class EditorController {
 
         if (params.action == "addRecordScreen") {
             def speciesList = SpeciesList.findByDataResourceUid(params.id)
+            if (!isCurrentUserEditorForList(sl)) {
```

Comment:
Invalid argument `sli` passed in to isCurrentUserEditorForList. Should be `speciesList`

Reply:
Updated.